packages/fieldset/package.json

```diff
@@ -37,6 +37,7 @@
     "@lion/validate": "0.6.4"
   },
   "devDependencies": {
+    "@lion/button": "0.5.2",
```

Comment:
this will mean that an update of button will trigger a release of fieldset - which will trigger, form, checkbox-group, radio-group, select, ...

could we move the test into the `form-system` package?